tools/create-test-data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import python_freeipa
+from python_freeipa import Client
+import random
```

Comment:
Traditionnaly, imports from the std library come first, and are separated from other imports by a blank line.